storage-units/src/indexfile.rs

```diff
@@ -149,10 +161,11 @@ impl Index {
         let bloom_size = default_bloom_size(entries);
         let params = Params {
             bloom_size: bloom_size,
+            ordinal: ordinal,
         };
 
         write_size(&mut hdr_buf[0..4], bloom_size);
-        write_size(&mut hdr_buf[4..8], 0);
+        write_size(&mut hdr_buf[4..8], ordinal);
```

Comment:
It is ok to steal this unused but reserved 4-byte "field" in lookup header, or should I better add a new one. Note that extending the header size will require a change in reading logic in many places, if this field were prepared for something - maybe it might be better for me to take it now, cuz for you it should be way easier to add a new one and not mess something up later, when needed. I can add a new one myself too, but just thinking of risk/time handling )